wren-ui/migrations/20241226135712_remove_thread_sql.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = async function (knex) {
+  // drop foreign key constraint before altering column type to prevent data loss
+  await knex.schema.alterTable('thread_response', (table) => {
+    table.dropForeign('thread_id');
+  });
+  await knex.schema.alterTable('thread', (table) => {
+    table.dropColumn('sql');
+  });
+  await knex.schema.alterTable('thread_response', (table) => {
+    table.foreign('thread_id').references('thread.id').onDelete('CASCADE');
+  });
+};
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.down = async function (knex) {
+  await knex.schema.alterTable('thread_response', (table) => {
+    table.dropForeign('thread_id');
+  });
+  await knex.schema.alterTable('thread', (table) => {
+    table.text('sql').nullable();
+  });
+  await knex.schema.alterTable('thread_response', (table) => {
+    table.foreign('thread_id').references('thread.id').onDelete('CASCADE');
+  });
```

Comment:
Isn't it the same as the statements in `up` migration ?

```
ON DELETE { NO ACTION | CASCADE | SET NULL | SET DEFAULT }
```
I think the default is NO ACTION. You could alter back to default.